src/components/Holdings.tsx

```diff
@@ -82,54 +24,80 @@ export default function Holdings() {
+        <div className="flex flex-col items-center justify-center gap-8">
+          <Heading />
+          <TotalReserveHoldings />
+        </div>
+        <ReserveAssetGrid />
+      </article>
+    </>
+  );
+}
+
+const Heading = () => {
+  return (
+    <h2 className="mx-auto text-center font-fg text-[32px] font-medium">
+      Current Reserve Holdings
+    </h2>
+  );
+};
 
+const ReserveAssetGrid = () => {
+  const {
+    data: { celo, otherAssets },
+    isLoadingOther,
+    isLoadingCelo,
+  } = useHoldings();
+
+  return (
+    <section className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-4">
+      <Amount
+        iconSrc={"/assets/tokens/CELO.svg"}
+        context="Funds in on-chain Reserve contract and in custodyy"
+        loading={isLoadingCelo}
+        label={"CELO"}
+        units={celo.unfrozen.units + celo.custody.units}
+        value={celo.unfrozen.value + celo.custody.value}
+      />
+      {otherAssets
+        ?.filter(skipZeros)
+        ?.map((asset) => (
           <Amount
-            iconSrc={"/assets/tokens/CELO.svg"}
-            context="Funds in on-chain Reserve contract and in custody"
-            loading={isLoadingCelo}
-            label={celo.frozen.value > 0 ? "Unfrozen" : "CELO"}
-            units={celo.unfrozen.units + celo.custody.units}
-            value={celo.unfrozen.value + celo.custody.value}
+            iconSrc={`/assets/tokens/${asset.token}.svg`}
+            key={asset.token}
+            loading={isLoadingOther}
+            label={asset.token}
+            units={asset.units}
+            value={asset.value}
           />
+        ))}
+    </section>
+  );
+};
 
-          <Heading
-            className="mt-[30px]"
-            title="Non-Celo Crypto Assets"
-            gridArea="crypto"
-          />
-          {data?.otherAssets
-            ?.filter(skipZeros)
-            ?.map((asset) => (
-              <Amount
-                key={asset.token}
-                loading={isLoadingOther}
-                label={asset.token}
-                units={asset.units}
-                value={asset.value}
-              />
-            ))}
-        </div>
-        <PieChart
-          label={"Current Composition"}
-          slices={percentages}
-          isLoading={isLoadingCelo || isLoadingOther}
-        />
-      </Section>
-    </>
+const TotalReserveHoldings = () => {
+  const { isLoading, totalReserveValue } = useReserveTotals();
+
+  return (
+    <span
+      className={cn(
+        "inline-flex h-[58px] w-full items-center justify-between gap-5 rounded-md border-[1px] border-black bg-mento-mint p-4 font-fg text-[22px] md:h-[58px] md:w-[530px] md:gap-[90px] md:text-[26px]",
+      )}
+    >
+      <span>Total reserve holdings:</span>
+      {isLoading || !totalReserveValue ? (
+        <Skeleton className="h-[18px] w-[148px] bg-[#d8e9d0] md:h-[26px] md:w-[10.3rem]" />
+      ) : (
+        <span className="font-medium">{`${totalReserveValue.toFixed(2)} $`}</span>
```

Comment:
1. let's add comma separators here same as for all other $ amounts
2. Also, the $ is suffixed here, whereas in all subcomponents it's prefixed. Should use the same approach everywhere, probably prefer prefixed